Rules/AvoidUsingInternalURLs.cs

```diff
@@ -118,7 +113,12 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
                             }
                             if (!firstPartURL.Contains("."))
                             {
-                                isInternalURL = true;
+                                //Add a check to exclude potential SDDL format. Check if a string have four components separated by ":"
```

Comment:
This is the mininum format for SDDLs. So if you see a string that has owner/group/discretionary ACLs and security ACLs that it is a SDDL. I believe the ordering is required.
https://msdn.microsoft.com/en-us/library/windows/desktop/aa379570(v=vs.85).aspx

O:owner_sid
G:group_sid
D:dacl_flags(string_ace1)(string_ace2)... (string_acen)
S:sacl_flags(string_ace1)(string_ace2)... (string_acen)